user_research/reset_env.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
_not a blocker_
Given it's a kubernetes resource YAML, could we put it in `kubectl_deploy/reset_user_research_db.yaml`?

Reply:
sure thing happy to change this